README.rst

```diff
@@ -111,6 +111,14 @@ You may import the coding style from IntelliJ so that the coding style can be ap
 - Enable `google-java-format` for the JDBC project.
 - In the source code window, select **Code** -> **Reformat** to apply the coding style.
 
+Thin Jar
+========
+
+To build a thin jar run command:
+
+.. code-block:: bash
+
+    mvn clean verify -Dthin-jar -Dnot-self-contained-jar
```

Comment:
A sentence of explanation what is the difference between the profiles? Both seem to be similar.

Reply:
Added